frontend/src/pages/profile/components/user-activity/styles.module.scss

```diff
@@ -0,0 +1,88 @@
+.wrapper {
+  grid-area: activity;
+}
+
+.title {
+  margin: 30px 0 15px;
+}
+
+.activityWrapper {
+  display: flex;
+  max-width: 710px;
+  overflow-x: auto;
+}
+
+.months {
+  display: grid;
+  grid-template: auto / repeat(5, 1fr) repeat(2, 0.8fr);
+  padding-bottom: 6px;
+}
+
+.months > span {
+  justify-self: center;
+}
+
+.days {
+  display: grid;
+  grid-template: 27px repeat(7, 25px) / auto;
+  grid-template-areas: "." "mon" "." "wed" "." "fri" "." "sun";
+  padding-right: 6px;
+}
+
+.days > span:nth-child(1) {
+  grid-area: mon;
+}
+
+.days > span:nth-child(2) {
+  grid-area: wed;
+}
+
+.days > span:nth-child(3) {
+  grid-area: fri;
+}
+
+.days > span:nth-child(4) {
+  grid-area: sun;
+}
```

Comment:
Why not to just set class from corresponding span - they are hardcoded, so don't now why we should calculate childs